src/sentry/models/rollbackorganizationdata.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models.base import DefaultFieldsModelExisting, region_silo_model
+from sentry.db.models.fields.foreignkey import FlexibleForeignKey
+
+
+@region_silo_model
+class RollbackOrganizationData(DefaultFieldsModelExisting):
+    """
+    A model for storing organization data by year for rollback purposes
```

Comment:
Might modify this a bit to be more descriptive, since the Rollback name can be confusing:

```
Stores a summary of each organization's year-in-review information to power the 2024 Sentry Rollback.
```

Reply:
I don't know if you need `Data` in the table name. Tables always contain data :smile: